app/components/join-steps/step-two.js

```diff
@@ -22,30 +38,39 @@ export default class StepTwoComponent extends Component {
     this.isValid = this.args.isValid;
     this.setIsValid = this.args.setIsValid;
     this.setIsPreValid = this.args.setIsPreValid;
-    const validated =
-      validator(this.data.introduction, 1) &&
-      validator(this.data.skills, 1) &&
-      validator(this.data.college, 1) &&
-      validator(this.data.forFun, 100) &&
-      validator(this.data.funFact, 100);
+    const validated = this.isDataValid();
     localStorage.setItem('isValid', validated);
     this.setIsPreValid(validated);
   }
+  isDataValid() {
+    for (let field in this.data) {
+      if (!validator(this.data[field], LIMITS[field])) {
+        return false;
+      }
+    }
+    return true;
+  }
 
   @action inputHandler(e) {
     this.setIsPreValid(false);
+
     const setValToLocalStorage = () => {
       this.data = { ...this.data, [e.target.name]: e.target.value };
       localStorage.setItem('stepTwoData', JSON.stringify(this.data));
-      const validated =
-        validator(this.data.introduction, 1) &&
-        validator(this.data.skills, 1) &&
-        validator(this.data.college, 1) &&
-        validator(this.data.forFun, 100) &&
-        validator(this.data.funFact, 100);
-      this.setIsValid(validated);
-      localStorage.setItem('isValid', validated);
+
+      // Only validate the changed field
+      const field = e.target.name;
+      const isValid = validator(this.data[field], LIMITS[field]);
+      this.errorMessage = {
+        ...this.errorMessage,
+        [field]: isValid ? '' : `At least, ${LIMITS[field]} word(s) required`,
```

Comment:
Can we please show the remaining word count here from max count ?
![image](https://github.com/Real-Dev-Squad/website-www/assets/78433013/6dbcade1-ce19-4767-9d93-33e7abddab0a)